controllers/gslb_controller_test.go

```diff
@@ -339,8 +340,8 @@ func TestGslbCreatesDNSEndpointCRForHealthyIngressHosts(t *testing.T) {
 	err = settings.client.Get(context.TODO(), settings.request.NamespacedName, dnsEndpoint)
 	require.NoError(t, err, "Failed to load DNS endpoint")
 	got := dnsEndpoint.Spec.Endpoints
-	prettyGot := utils.ToString(got)
-	prettyWant := utils.ToString(want)
+	prettyGot := str.Format(got)
```

Comment:
Please consider renaming `Format()` function back to `ToString()` in order to resolve ambiguity.
In many languages and std libraries, format functions are universal, and provide string formatting according to provided format expression.
In this current case we are dealing with simplified conversion function without parameters, plus having its internal logic.
In this case old `ToString()` name perfectly serves the purpose.

Reply:
@somaritane , please see: [gopkg #7 ](https://github.com/AbsaOSS/gopkg/pull/7)